src/libsyncengine/jobs/network/login/abstractloginjob.cpp

```diff
@@ -71,7 +71,26 @@ bool AbstractLoginJob::handleError(std::istream &is, const Poco::URI &uri) {
     try {
         jsonError = jsonParser.parse(is).extract<Poco::JSON::Object::Ptr>();
     } catch (Poco::Exception &exc) {
-        LOG_DEBUG(_logger, "Reply " << jobId() << " received doesn't contain a valid JSON error: " << exc.displayText().c_str());
+        LOG_WARN(_logger, "Reply " << jobId() << " received doesn't contain a valid JSON error: " << exc.displayText().c_str());
+
+        // Try to parse as string
+        std::stringstream errorStream;
+        errorStream << "Reply " << jobId();
+
+        std::string str(std::istreambuf_iterator<char>(is), {});
```

Comment:
A bit out of scope, but the variable name `is` is particularly non-suggestive.

Reply:
I can understand that. Do you have a better name in mind? On https://en.cppreference.com/w/cpp/io/basic_istream they are quiet stingy too in the of characters in the variable name

Reply:
Done